tests/test_contracts.py

```diff
@@ -88,17 +108,30 @@ def test_scaffolder_generate(scaffolder):
 
 
 @responses.activate
-def test_scaffolder_generate_openaea_contract(scaffolder, test_filesystem):
+def test_scaffolder_generate_openaea_contract(scaffolder, test_filesystem, tmp_path):
```

Comment:
I dont think this test should be changed as it is now creating a nested fiel structure.

Reply:
will revert